parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -25,6 +25,47 @@ installDeps() {
     fi
 }
 
+installKataDeps() {
+    if [[ $OS_VERSION == "2.0" ]]; then
+      for dnf_package in cargo opa parted qemu-img moby-runc python3-devel python3-pip kernel-mshv cloud-hypervisor kata-containers moby-containerd-cc mshv-bootloader mshv-linuxloader mshv; do
```

Comment:
Please also wait for the loader-related changes in my branch to be merged into this branch before merging.

Reply:
see: https://github.com/Camelron/AgentBaker/commits/cameronbaird/kata-cc-image-curation

Reply:
Got it.